### DISCUSSION TITLE
[helm] support minio distributed mode

### DISCUSSION THREAD

**Discussion by josh-ferrell** [airbytehq/airbyte#33139] [Connector Ideas and Features]
### What area the feature impact?

Helm Chart

### Relevant Information

Being able to deploy minio in distributed mode to increase the number of replicas via the airbyte helm values would be helpful for reliability objectives.

Minio Upstream Chart: [minio/helm/minio/templates/statefulsets.yaml](https://github.com/minio/minio/blob/master/helm/minio/templates/statefulset.yaml)

**Comment by marcosmarxm** [comment]
Thanks @josh-ferrell I moved the issue to a discussion. It is easier to get feedback (votes) from other community members.

[a comment by an automated account is not shown]
